package.json

```diff
@@ -80,6 +81,7 @@
     "lodash.has": "^4.5.2",
     "lodash.isempty": "^4.4.0",
     "lodash.isequal": "^4.5.0",
+    "luxon": "^3.5.0",
```

Comment:
What is this pkg being used for? Is it still being used or can it be removed?

Reply:
tabulator lib we are using for tables requires luxon and type/luxon for date sorting 
Dependency Required
You will need to include the luxon.js library to use this sorter